modules/ROOT/pages/reference/procedures.adoc

```diff
@@ -318,6 +318,16 @@ In 4.2, signature changed to `db.listLocks() :: (mode :: STRING, resourceType ::
 | label:yes[]
 | label:new[Introduced in 5.23] label:server-management[`SERVER MANAGEMENT` privilege only]
 
+| xref:reference/procedures.adoc#procedure_dbms_cluster_recreateDatabase[`dbms.cluster.recreateDatabase()`]
+| label:no[]
+| label:yes[]
+| label:new[Introduced in 5.24]
```

Comment:
Would it make sense to include the required privileges here as well? You need to have the same privileges as for dropping and creating a database.

Reply:
Usually we add the 'Admin only' label if you need `Admin` privileges.

Reply:
Just above this procedure, there are a bunch of procedures with "label:server-management[`SERVER MANAGEMENT` privilege only]". I thought that might be in the same realm as DROP and CREATE database privileges?

Reply:
yep, I know that we have several procedures with this label `SERVER MANAGEMENT`. But in fact this was an exception.
So, you mean this type of privileges - https://neo4j.com/docs/operations-manual/current/authentication-authorization/dbms-administration/#access-control-dbms-administration-database-management, don't you?

Reply:
or we can talk about the privileges on another page, where we'll explain how to use this procedure

Reply:
Yes 😄  Is that not done?

Reply:
sorry, what do you mean? 😁